### ISSUE TITLE
Repeat order.

### ISSUE THREAD

**Issue by AliYoussef97** [lucidrains/rotary-embedding-torch#23]
Hello,

Thank you for the amazing work!

I had a brief question, shouldn't `(n r)` in repeat be `(r n)` [here](https://github.com/lucidrains/rotary-embedding-torch/blob/783d17820ac1e75e918ae2128ab8bbcbe4985362/rotary_embedding_torch/rotary_embedding_torch.py#L277). As (r n)!=(n r), as `(r n)` would be equivalent to `freqs = torch.cat((freqs, freqs), dim=-1)`. I might be compeltely wrong though, and order does not matter.

Thank you!
